src/snapred/backend/dao/request/NormalizationRequest.py

```diff
@@ -27,3 +30,5 @@ class NormalizationRequest(BaseModel, extra="forbid"):
     )
     nBinsAcrossPeakWidth: int = Config["calibration.diffraction.nBinsAcrossPeakWidth"]
     fwhmMultipliers: Pair[float] = Pair.model_validate(Config["calibration.parameters.default.FWHMMultiplier"])
+
+    continueFlags: Optional[ContinueWarning.Type] = ContinueWarning.Type.UNSET
```

Comment:
I wonder if these should live on some parent request object

Reply:
**I also think that they should.**  There's a TODO somewhere about adding them to `SNAPRequest`.  However, the problem right now is that we don't derive most of our requests from `SNAPRequest`, which _might_ _also_ be a good idea.